.github/workflows/format.yaml

```diff
@@ -39,5 +39,5 @@ jobs:
 
       - name: Check formatting
         run: |
-          shfmt -l -d .
+          shfmt -i 2 -ci -l -d .
```

Comment:
cc: @ChristianChiarulli

Reply:
@kylo252 if we are changing this, we might wanna change the default formatting on `core/lang/sh.lua` as well
https://github.com/ChristianChiarulli/LunarVim/blob/0064b446a02be13dba0f16f084e073b347716710/lua/lang/sh.lua#L13-L17

Reply:
I'm gonna move that shfmt style commit to a separate PR since it's affecting a few files already.

Reply:
sounds good to me :+1: